### PR TITLE
Added database

### PR DESCRIPTION
**Pull request by ovekyc**

### CODE REVIEW

File: app/core/database.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      });
+    return;
+  }
+  fs.appendFileSync(logPath, clip);
+}
+
+const db = levelup('log.db', {valueEncoding: 'json'});
+export async function store(cb) {
+  const clip = clipboard.readText();
+  console.log(clip); // eslint-disable-line no-console
```
Comment (yuda1124) [inline app/core/database.js]: why don't you delete this?

File: app/core/database.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+  const tranGoogle = await googleTranslate(clip);
+  const tranGlosbe = await glosbeTranslate(clip);
+  console.log(tranGoogle);
+  console.log(tranGlosbe);
```
Comment (yuda1124) [inline app/core/database.js]: console.log

File: app/core/database.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+import Contents from './content';
+
+// TODO: remove, for backward compatibility
+import fs from 'fs';
+import setting from '../setting.json';
+import {SERVICE} from './constants';
+import {logPath} from './main';
+export function saveContents() {
+  const clip = clipboard.readText();
+  console.log(clip); // eslint-disable-line no-console
```
Comment (yuda1124) [inline app/core/database.js]: console.log

File: app/core/main.js [hunk 4]
```diff
@@ -1,46 +1,39 @@
...
-    service(clip)
-      .then(translated => `${clip} => ${translated}\n`)
-      .then(contents => {
-        fs.appendFileSync(logPath, contents);
-      });
-    return;
-  }
-  fs.appendFileSync(logPath, clip);
+function notifyDone(contents) {
+  console.log(contents);
```
Comment (yuda1124) [inline app/core/main.js:17]: it will be changed popup or something.
how about add todo!

File: app/core/main.js [hunk 5]
```diff
@@ -1,46 +1,39 @@
...
+}
+
+function notifyErr(err) {
```
Comment (yuda1124) [inline app/core/main.js:20]: ditto

File: app/core/main.js (hunk 4 above)
Comment (ovekyc) [reply app/core/main.js:17]: great

**Review by yuda1124 — APPROVED** [review]

**Pull request merged**